lib/src/formatter.dart

```diff
@@ -173,9 +173,7 @@ class SimpleFormatter implements ReportFormatter {
     var tableWidth = max(_summaryLength, longest + largestCountGuess);
     var pad = tableWidth - longest;
     var line = ''.padLeft(tableWidth, '-');
-    out.writeln(line);
-    out.writeln('Counts');
-    out.writeln(line);
+    out..writeln(line)..writeln('Counts')..writeln(line);
```

Comment:
I don't know that it's always useful to have cascades on separate lines, but yes, if they were formatted that way I wouldn't have a problem with using cascades in this case. :-)